### PR TITLE
floorp-portable: refactor

### PR DESCRIPTION
**Pull request by regularentropy**
This PR improves readability and refactors some parts of the code.

### CODE REVIEW

**Comment by Ryosuke-Asano** [comment]
WOW.

@creeper-0910

File: fileInUse_windows.go
Comment (creeper-0910) [inline fileInUse_windows.go:1]: Error handling on line 21 may be necessary.
Sorry if I am wrong.

Comment (regularentropy) [reply fileInUse_windows.go:1]: > Error handling on line 21 may be necessary. Sorry if I am wrong.

It was only usable with the crypto/rand library and the rand.Reader variable.

After the simplification of the algorithm, I found this check unnecessary, as the chance of err occurring here is minimal.

Comment (creeper-0910) [reply fileInUse_windows.go:1]: I understand. Thank you!

**Review by creeper-0910 — APPROVED** [review]

**Pull request merged**